feature-libs/checkout/b2b/occ/config/default-occ-checkout-b2b-config.ts

```diff
@@ -1,12 +1,22 @@
+// We need these imports for augmentation of OccEndpoints to be picked up
+import { CheckoutOccEndpoints } from '@spartacus/checkout/base/occ';
```

Comment:
Reminder for renaming services to facades: https://github.com/SAP/spartacus/pull/14165#discussion_r749938690

Reply:
Done